### PR TITLE
Don't throw `Domain not found` when trying to `now alias` to a new domain

### PR DESCRIPTION
**Pull request by matheuss**
Before
```
▲ now-cli at master ✔ now alias gifs-xfylwzhmgg.now.sh zeit.com
> zeit.com is a custom domain.
> Verifying the DNS settings for zeit.com (see https://zeit.world for help)
> Error! Domain name not found
```

After
```
▲ now-cli at fix/alias-new-domain ✔ now alias gifs-xfylwzhmgg.now.sh zeit.com           
> zeit.com is a custom domain.
> Verifying the DNS settings for zeit.com (see https://zeit.world for help)
[...]
```

### CODE REVIEW

**Review by OlliV — APPROVED** [review]

File: lib/alias.js [hunk 1]
```diff
@@ -670,9 +670,20 @@ module.exports = class Alias extends Now {
...
+        _domainInfo = await this.getDomain(_domain)
+      } catch (err) {
+        if (err.status === 404) {
+          // It's ok if the domain was not found – we'll add it when creating
+          // the alias
+        } else {
+          throw err
+        }
+      }
+      const domainInfo = _domainInfo && !_domainInfo.error
```
Comment (TooTallNate) [inline lib/alias.js]: Shouldn't we do something with `_domainInfo.error` like relay it to the user?

Comment (matheuss) [reply lib/alias.js]: @TooTallNate we handle that somewhere – the user will see proper errors:
```
now alias gifs-zjduxbfjas.now.sh vigour.io
> vigour.io is a custom domain.
> Verifying the DNS settings for vigour.io (see https://zeit.world for help)
> Error! You don't have access to "vigour.io"
```

**Comment by matheuss** [comment]
Will fix the prettier stuff and fix it ASAP 👍

**Comment by OlliV** [comment]
@matheuss status?

**Pull request merged**